pages/services-indoor-positioning-system-development.js

```diff
@@ -213,10 +213,10 @@ const ServicesIndoorPositioningDevelopment = () => {
 
           <h2>THE THREE ESSENTIAL FEATURES</h2>
 
-          <div className="services-ips-wdwo__list">
+          <div className="page__list">
               {
                 essentialFeaturesList.map(({title, description}) => (
-                  <div className="services-ips-wdwo__item">
+                  <div className="services-ips-wdwo__item page__item">
```

Comment:
@wkornks , not sure we are using BEM.
@soloviovmax , are we? What's the best approach here?